### PR TITLE
LPS-175202 Clear object action Ids at the end of deleteObjectEntry (as done in addObjectEntry)

### PR DESCRIPTION
**Pull request by devpedrotavares**
Related issue: [LPS-175202](https://issues.liferay.com/browse/LPS-175202)

### CODE REVIEW

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectActionLocalServiceTest.java [hunk 1]
```diff
@@ -568,7 +570,54 @@ public void testAddObjectActionWithMoreThanOneObjectEntry()
...
+
+		ObjectEntry objectEntry3 = _objectEntryLocalService.addObjectEntry(
+			TestPropsValues.getUserId(), 0,
+			_objectDefinition.getObjectDefinitionId(),
+			HashMapBuilder.<String, Serializable>put(
+				"firstName", "John"
+			).build(),
+			ServiceContextTestUtil.getServiceContext());
+
+		ObjectEntry objectEntry4 = _objectEntryLocalService.addObjectEntry(
```
Comment (carolmariaabb) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectActionLocalServiceTest.java:602]: @devpedrotavares can we remove this empty line here?

**Review by carolmariaabb — CHANGES_REQUESTED** [review]

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectActionLocalServiceTest.java (hunk 1 above)
Comment (devpedrotavares) [reply modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectActionLocalServiceTest.java]: Yes. Just did it!

**Comment by devpedrotavares** [comment]
ci:test:sf

**Review by carolmariaabb — APPROVED** [review]
LGTM!

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Pull request closed without merging**